int3face/assetlist.json

```diff
@@ -0,0 +1,205 @@
+      "name": "Bitcoin",
+      "display": "btc",
+      "symbol": "BTC",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "bitcoin",
+            "base_denom": "sat"
+          },
+          "provider": "Int3face"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3btc.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3btc.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3btc.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3btc.svg",
+          "theme": {
+            "primary_color_hex": "#3d3d3d"
+          }
+        }
+      ],
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Bitcoin-Cash on Int3face",
+      "denom_units": [
+        {
+          "denom": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/bitcoin-cash-bch",
+          "exponent": 0
+        },
+        {
+          "denom": "bch",
+          "exponent": 8
+        }
+      ],
+      "base": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/bitcoin-cash-bch",
+      "name": "Bitcoin Cash",
+      "display": "bch",
+      "symbol": "BCH",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "bitcoin-cash",
+            "base_denom": "sat"
+          },
+          "provider": "Int3face"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3bch.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3bch.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3bch.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3bch.svg",
+          "theme": {
+            "primary_color_hex": "#3d3d3d"
+          }
+        }
+      ],
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Dogecoin on Int3face",
+      "denom_units": [
+        {
+          "denom": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/dogecoin-doge",
+          "exponent": 0
+        },
+        {
+          "denom": "doge",
+          "exponent": 8
+        }
+      ],
+      "base": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/dogecoin-doge",
+      "name": "Dogecoin",
+      "display": "doge",
+      "symbol": "DOGE",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "dogecoin",
+            "base_denom": "koinu"
```

Comment:
The dogecoin base denom is currently defined as 'shibe', as per https://github.com/cosmos/chain-registry/blob/master/_non-cosmos/dogecoin/assetlist.json#L19